lib/signal-list-view.coffee

```diff
@@ -29,6 +29,11 @@ class SignalListView extends SelectListView
 
         @wsKernelCommands = [
             {
+                name: 'Rename'
```

Comment:
I have started cleaning up `SignalListView` some time ago. I'll address this in this PR.